			   READLINE PATCH REPORT
			   =====================

Readline-Release: 8.2
Patch-ID: readline82-003

Bug-Reported-by:	Stefan Klinger <readline-gnu.org@stefan-klinger.de>
Bug-Reference-ID:
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-readline/2023-08/msg00018.html

Bug-Description:

Patch (apply with `patch -p0'):

The custom color prefix that readline uses to color possible completions
must have a leading `.'.

*** ../readline-8.2-patched/colors.c	2021-12-08 11:38:25.000000000 -0500
--- colors.c	2023-08-28 16:40:04.000000000 -0400
***************
*** 74,78 ****
  static void restore_default_color (void);
  
! #define RL_COLOR_PREFIX_EXTENSION	"readline-colored-completion-prefix"
  
  COLOR_EXT_TYPE *_rl_color_ext_list = 0;
--- 74,78 ----
  static void restore_default_color (void);
  
! #define RL_COLOR_PREFIX_EXTENSION	".readline-colored-completion-prefix"
  
  COLOR_EXT_TYPE *_rl_color_ext_list = 0;
*** ../readline-8.2/patchlevel	2013-11-15 08:11:11.000000000 -0500
--- patchlevel	2014-03-21 08:28:40.000000000 -0400
***************
*** 1,3 ****
  # Do not edit -- exists only for use by patch
  
! 2
--- 1,3 ----
  # Do not edit -- exists only for use by patch
  
! 3
